main.cpp

```diff
@@ -389,5 +419,11 @@ int main()
     guitarAndPedal.modifyPickups(3);
     guitarAndPedal.turnRatioKnob(6);
 
+    compy.changeNumDataTypesThis(6);
+    compy.changeNumOperatorTypesThis(11);
+
+    guitarAndPedal.modifyPickupsThis(5);
+    guitarAndPedal.turnRatioKnobThis(8);
+
```

Comment:
Your `main()` has no `std::cout` statements that are being replicated by these member functions.
review the example again.
- [ ] revise